elide-core/src/test/java/com/yahoo/elide/utils/coerce/converters/ISO8601DateSerdeTest.java

```diff
@@ -67,6 +68,14 @@ public void testSQLTimestampDeserialization() throws Exception {
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testDateToSQLTimestampDeserialization() throws Exception {
+        ISO8601DateSerde serde =
+                new ISO8601DateSerde("yyyy-MM-dd", TimeZone.getTimeZone("UTC"), java.sql.Timestamp.class);
+        java.sql.Timestamp expected = new Timestamp(0);
+        java.sql.Timestamp actual = (java.sql.Timestamp) serde.deserialize(new Date(0));
```

Comment:
Timestamp already imported.
```suggestion
        Timestamp expected = new Timestamp(0);
        Timestamp actual = (Timestamp) serde.deserialize(new Date(0));
```